src/System.IdentityModel.Tokens/Utility.cs

```diff
@@ -65,5 +65,25 @@ internal static string SerializeAsSingleCommaDelimitedString(IEnumerable<string>
 
             return sb.ToString();
         }
+
+        public static bool IsHttps(string address)
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                return false;
+            }
+
+            return address.StartsWith("https");
```

Comment:
A quick test showed that "HTTPS://" would return false. 
"HTTPSS://" would return true.

What is the other overload?